pyproject.toml

```diff
@@ -41,7 +41,7 @@ dependencies = [
     "openpyxl>=2.6",
     "pandas>=0.21",
     "pyyaml>=5.3",
-    "scipy>=1.2",
+    "scipy>=1.7",
```

Comment:
Just a heads-up: we have to support scipy 1.5.3 at minimum in this package due to it being included in RMS environments. All RMS versions below 14.2.2 (which is the newest and still not widely adopted by assets) lock scipy to 1.5.3, a vendor limitation we can't relax

Reply:
Thanks!
Do you know if `fmudesign` is used from within RMS?

Reply:
To my knowledge it's not used directly but @jcrivenaes may need to correct me

Reply:
If not, could we consider moving to a separate repo?

Reply:
> To my knowledge it's not used directly but @jcrivenaes may need to correct me

Not to my knowledge either

Reply:
> If not, could we consider moving to a separate repo?

I think this is completely fine if Jan and Trine think so as well, so long as we hang a link from the design docs here to its new location

Reply:
OK with me

Reply:
Yes, I think that should be ok. Quite sure it's not used within RMS.